package.json

```diff
@@ -1,16 +1,10 @@
 {
-  "name": "browserd",
+  "name": "browsered",
```

Comment:
is it common lerna best practice to include a package.json in the root?

Reply:
also this name is incorrect :)

Reply:
Yep I was working according to this structure from the Lerna Docs
`my-lerna-repo/
  package.json
  packages/
    package-1/
      package.json
    package-2/
      package.json`